File: cc practice/cybv.py
t = int(input())

for i in range(t):
    n, k = input().split()
    n, k = int(n), int(k)

    c = k - k%n
    result = c/n
    print(int(result))
    '''
    explanation:
    k/n number of guns will be given to all
    the remaining guns(k%n) will be given to few kids
    but we are intrested in least
    so CAKEWALK!!
    I messed up :(
    '''

File: cc practice/cybv.c
#include <stdio.h>

int main(void)
{
    int t;
    scanf("%i", &t);

    for(int i=0; i < t; i++) {
        int n, k;
        scanf("%i", &n);
        scanf("%i", &k);

        int c = k - k%n;
        int result = c/n;

        printf("%i", result);
    }
}